packages/ui/src/styles/globals.css

```
@@ -1,8 +1,83 @@
 @import "tailwindcss";
 @import "./elevation.css";
 @import "./animation.css";
+@import "./color-tokens.css";
+@import "./system-tokens.css";
+@import "./typo-tokens.css";
 @source "../components/**/*.{ts,tsx}";
 
+@font-face {
+  font-family: "Pretendard";
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Pretendard" (font-family-name-quotes)

(font-family-name-quotes)